backend/grants/admin.py

```diff
@@ -198,19 +199,37 @@ def wrapper(modeladmin, request, queryset):
     return wrapper
 
 
+def validate_visa_page_public(func):
+    """
+    Ensure the '/visa/' page exists and is public on the conference website.
+    """
+
+    @wraps(func)
+    def wrapper(modeladmin, request, queryset):
+        site = Site.find_for_request(request)
```

Comment:
this is probably not going to work, as we have a middleware hardcoding the allowed_host to pycon.it, it won't find the right site